interfaces/cython/cantera/yaml2ck.py

```diff
@@ -0,0 +1,832 @@
+# This file is part of Frhodo. Copyright © 2020, UChicago Argonne, LLC
+# and licensed under BSD-3-Clause. See License.txt in the top-level
+# directory for license and copyright information.
+
+# Adapted from Kyle Niemeyer's pyMARS Jul 24, 2019
```

Comment:
I believe this section needs to be resolved before going any further.

Reply:
Ah, sorry. In any case, MIT is a similar permissive style license as BSD, so it doesn't pose a problem.